masonry/src/widgets/button.rs

```diff
@@ -419,10 +400,22 @@ mod tests {
 
         let grid = Grid::with_dimensions(2, 2)
             .with_spacing(40.0)
-            .with_child(Button::new("A").with_auto_id(), GridParams::new(0, 0, 1, 1))
-            .with_child(Button::new("B").with_auto_id(), GridParams::new(1, 0, 1, 1))
-            .with_child(Button::new("C").with_auto_id(), GridParams::new(0, 1, 1, 1))
-            .with_child(Button::new("D").with_auto_id(), GridParams::new(1, 1, 1, 1));
+            .with_child(
+                Button::new(Label::new("A").with_auto_id()).with_auto_id(),
+                GridParams::new(0, 0, 1, 1),
+            )
+            .with_child(
+                Button::new(Label::new("B").with_auto_id()).with_auto_id(),
+                GridParams::new(1, 0, 1, 1),
+            )
+            .with_child(
+                Button::new(Label::new("C").with_auto_id()).with_auto_id(),
+                GridParams::new(0, 1, 1, 1),
+            )
+            .with_child(
+                Button::new(Label::new("D").with_auto_id()).with_auto_id(),
+                GridParams::new(1, 1, 1, 1),
+            );
```

Comment:
For instance, you could keep these tests fairly short by using a `Button::with_text` constructor.